src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -345,6 +345,7 @@ public static bool IsTypeDeclaration(SyntaxKind kind)
                 case SyntaxKind.InterfaceDeclaration:
                 case SyntaxKind.DelegateDeclaration:
                 case SyntaxKind.EnumDeclaration:
+                case SyntaxKind.RecordDeclaration:
```

Comment:
>case SyntaxKind.RecordDeclaration: [](start = 16, length = 34)

Please make sure we have targeted tests for all modified APIs in this file.

Reply:
I looked around and didn't find any tests for these APIs, it usually looks like we rely on the parsing tests to validate that contextual tokens are properly lexed. Do you want me to add new tests?

Reply:
> Do you want me to add new tests?

At least add tests for those that you missed and existing tests didn't catch the fact, or add a note to Test Plan to follow up

---
In reply to: [435439309](https://github.com/dotnet/roslyn/pull/44841#discussion_r435439309) [](ancestors = 435439309)